src/List/ListItem.js

```diff
@@ -676,16 +687,16 @@ class ListItem extends Component {
           simpleLabel ? this.createLabelElement(styles, contentChildren, other) :
           disabled ? this.createDisabledElement(styles, contentChildren, other) : (
             <EnhancedButton
-              containerElement="span"
               {...other}
+              containerElement={containerElement}
```

Comment:
@lourd That's going to work the same, yes. My point was just around code readability.